cli/root.go

```diff
@@ -40,11 +44,27 @@ var (
 	timeout         time.Duration
 	httpConfigFile  string
 	versionCheck    bool
+	featureFlags    string
 
 	configFiles = []string{os.ExpandEnv("$HOME/.config/amtool/config.yml"), "/etc/amtool/config.yml"}
 	legacyFlags = map[string]string{"comment_required": "require-comment"}
 )
 
+func initMatchersCompat(_ *kingpin.ParseContext) error {
```

Comment:
This is where the switching of the feature happens. I needed to use `app.Action` so it would run after Kingpin had computed the flags but before running any commands.